modules/navier_stokes/test/tests/finite_volume/materials/flow_diode/tests

```diff
@@ -1,13 +1,54 @@
 [Tests]
+  issues = '#20695'
+  design = 'NSFVFrictionFlowDiodeMaterial.md'
+
   [friction_flow_diode]
     type = Exodiff
-    issues = '#20695'
-    design = 'NSFVFrictionFlowDiodeMaterial.md'
     requirement = 'The system shall be able to model a simple friction-based volumetric flow diode'
     input = 'friction.i'
     exodiff = friction_out.e
     ad_indexing_type = 'global'
     # see #21185
     allow_warnings = true
   []
+
+  [controlled]
+    requirement = 'The system shall be able to control a simple friction-based volumetric flow diode'
+    [time]
+      type = CSVDiff
+      input = 'transient_operation.i'
+      csvdiff = 'time_based.csv'
+      cli_args = 'Controls/active=time_based Outputs/file_base=time_based'
+      ad_indexing_type = 'global'
+      detail = 'with a simple time-based turn-on criterion,'
+      # see #21185
+      allow_warnings = true
```

Comment:
What's warning?

Reply:
https://github.com/idaholab/moose/issues/21185

Reply:
If there's a warning, why is the issue marked as closed?

Reply:
It wasn't clear to me from any of the text in the issue nor in the pull request titles that referenced the issue that a warning was the goal or being added. If the pull request closing an issue (in this case titled "More misc fixups") doesn't make clear why it's closing the issue, then a little more text explaining why we regard the issue as closed would be appreciated.

Reply:
The PR was about fixing the bad code that was in place of this warning.
I only have to reference issues in commits and PRs, not explain each commit in the text again. If the reviewer finds it unclear what each commit is doing, they are welcome to ask me to reword the message.

Reply:
I just found the relevant commit on the issue. Things are clear to me now I think. So are we just getting lucky in our CI that our distributed mesh runs don't distribute the sideset across different procs?

Reply:
I think we are lucky that the MG actually works with distributed meshes, but I have no idea why.

My guess is that the MG gets the nodeset right, then this gets converted to a sideset automatically ??

I m not sure but someone flagged this MG as not working with distributed meshes and I didnt have the expertise to overrule their opinion.

Reply:
I see. Understood. So it sounds like this test will likely continue to work even if the sideset gets partitioned across process boundaries.